config/docker/qemu/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM debian:bullseye
 
-RUN apt-get update && \
-	apt-get install --yes --no-install-recommends \
+RUN echo deb http://deb.debian.org/debian bullseye-backports main \
+	> /etc/apt/sources.list.d/backports.list
+
+RUN apt-get update
+
+RUN apt-get install -t bullseye-backports --yes --no-install-recommends \
```

Comment:
Please keep `update` and `install` on the same line. This is to avoid bad surprises with how Docker layer caching works: if the list of packages gets updated, `apt-get update` won't be run and the install might fail.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get